application/features/email.feature

```diff
@@ -205,11 +204,38 @@ Feature: Email
       | to send            | does exist     | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does exist      | delete_mfa   | should NOT  |
       | to send            | does exist     | does exist      | delete_mfa   | should NOT  |
-      | to send            | does NOT exist | does NOT exist  | delete_mfa   | should      |
       | NOT to send        | does NOT exist | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
       | NOT to send        | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
 
+
+  Scenario Outline: When to send an email after a verified u2f mfa option has been deleted.
+    Given we are configured to send mfa disabled emails
+      And we are configured to send mfa option removed emails
+      And no mfas exist
+      And a user already exists
+      And a verified u2f mfa option used to exist
+      And the latest mfa event type was delete_mfa
+    When I check if a mfa <optionRemovedOrDisabled> email should be sent
+    Then I see that a mfa <optionRemovedOrDisabled> email <shouldOrNot> be sent
+
+    Examples:
+      | optionRemovedOrDisabled | shouldOrNot |
+      | option removed          | should NOT  |
```

Comment:
I think there's a typo here: when we have just deleted a verified MFA option, then we _should_ send the "MFA option removed" email. Right?

Reply:
When it's the last mfa option of that type, we only send the mfa disabled email and not the mfa option removed email.

Reply:
I don't think we differentiate by MFA type when deciding whether to sent the "MFA option removed" vs. the "MFA disabled" emails: if they have _any_ remaining MFA options, then they have simply removed an MFA option; if they don't have any MFA options remaining, then they have disabled MFA.

Reply:
Never mind, you're right. I was overlooking that this was testing which email to send when the _last_ verified MFA option is deleted.